docs/api-irma-server.md

```diff
@@ -147,3 +162,102 @@ Also returns a session result JWT, but one whose structure is the same as the se
 ### `GET /publickey`
 
 If a JWT private key was [provided in the configuration of the `irma server`](irma-server.md#signed-jwt-session-results), then this returns the corresponding public key in PEM with which the server's session result JWTs returned by [`GET /session/{token}/result-jwt`](#get-session-token-result-jwt) and [`GET /session/{token}/getproof`](#get-session-token-getproof) can be verified.
+
+---
+
+## API reference `/irma` endpoints
+The `/irma` endpoints of your IRMA server have to be publicly available to the internet. Most of the endpoints
+behind the `/irma` prefix are exclusively used by the
+[irmaclient](https://github.com/privacybydesign/irmago/tree/master/irmaclient)/[IRMA app](irma-app.md).
+We do not document those endpoints here, because these are only relevant for
+irmaclient developers. We only document the endpoints that are relevant for [frontend libraries](irma-frontend.md).
+
+Note that all endpoints below use the `/irma/session/{clientToken}/...` prefix. This prefix exactly
+matches the `u` field from the `sessionPtr` object, so the location of `/irma` endpoints can also be derived doing
+`{sessionPtr.u}/...`. A `sessionPtr` can be obtained using the [`POST /session`](#post-session) requestor endpoint.
+
+---
+
+### `DELETE /irma/session/{clientToken}`
+Behaves exactly the same as the [delete endpoint for requestors](#delete-session-token), but uses the
+client token from the `sessionPtr` instead of the requestor token.
```

Comment:
Yeah I agree that it should be named the same everywhere to prevent confusion. The difficult part in the explanation will be to explain that the requestor token is marked as `token` in the JSON, so a little bit of confusion will always remain 😅